docs/station.md

```diff
@@ -0,0 +1,49 @@
+# station (Private Beta)
+A hybrid station has the capability to trigger a scan within your Virtual Private Cloud (VPC) environment based on the inputs provided through the HCP UI.
+The `station` command is used to fetch scan jobs from [HCP](https://portal.cloud.hashicorp.com/)and execute them in your VPC. It will scan any available jobs, otherwise it will sleep and wait for a job to be queued in HCP.
+
+## Setting Up
+In order to run `station` in your environment there are 3 configuration steps required in HCP and your local environment.
+
+### Create a Service Principal
+**Note** Does the type of SP matter in this case?
+You will need to log into the HCP web portal to create your Service Principals. You can follow the HCP docs to create your [Project Service Principals](https://developer.hashicorp.com/hcp/docs/hcp/admin/iam/service-principals#project-level-service-principals-1). You will need to create the Service Principals at Admin level, more information about this level contained in [these](https://developer.hashicorp.com/hcp/docs/hcp/admin/iam/users#project-role) HCP docs.
+
+### Create a Station in the HCP Portal
+> ***Note:*** Before you create a station, there is currently now way to delete stations via the UI
+
+Now, you will need to initiate a station in the HCP web portal. If it's your first time using `Vault Radar` in HCP, you will need to configure a data source of your choice in order to get the `Station` tab view. This data source will not be used for running the station at all. This is a known UX issue that will be fixed in a future iteration.
+
+Once you have the `Station` tab view, you can start working through the station creation flow. This will give you a Station ID you will use in your local environment.
+
+### Configure your Local Environment
```

Comment:
I think it would be good to call out that `HCP_CLIENT_ID` and `HCP_CLIENT_SECRET` are from the above service principal key creation, and also that `HCP_PROJECT_ID` will be provided during the setup flow